src/main/java/frc/trigon/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -100,6 +101,8 @@ private static void configureMasterMotor() {
         MASTER_MOTOR.registerSignal(TalonFXSignal.STATOR_CURRENT, 100);
         MASTER_MOTOR.registerSignal(TalonFXSignal.TORQUE_CURRENT, 100);
         MASTER_MOTOR.registerSignal(TalonFXSignal.MOTOR_VOLTAGE, 100);
+        MASTER_MOTOR.registerSignal(TalonFXSignal.REVERSE_LIMIT, 100);
+        MASTER_MOTOR.registerSignal(TalonFXSignal.FORWARD_LIMIT, 100);
```

Comment:
There aren't any limits on this motor, only the reverse limit of the climbers plugged into these motors